src/plugins/jartic/datasource.py

```
@@ -0,0 +1,268 @@
+
+        self._sensor_cache[location.id] = sensors
+        return sensors
+
+    async def stream_measurements(
+        self,
+        sensor: Sensor,
+        start_date: Optional[datetime] = None,
+        end_date: Optional[datetime] = None
+    ) -> AsyncIterator[Measurement]:
+        if start_date is None:
+            start_date = datetime(2023, 1, 1)
```

Ruff (0.12.2)

`datetime.datetime()` called without a `tzinfo` argument

(DTZ001)